src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  selectedPostId: number;
+  handleAddComment: (
+    postId: number,
+    name: string,
+    email: string,
+    body: string,
+  ) => void;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  selectedPostId,
+  handleAddComment,
+}) => {
+  const [name, setName] = useState('');
+  const [isNameError, setIsNameError] = useState(false);
+  const [email, setEmail] = useState('');
+  const [isEmailError, setIsEmailError] = useState(false);
+  const [body, setBody] = useState('');
+  const [isBodyError, setIsBodyError] = useState(false);
```

Comment:
My assumption it's better to have 2 states instead of 6. One for fields 
```
const [form, setForm] = useState({
  name: '',
  ....
});
```
second for error message< by default it's empty string
`const [errorMsg, setErrorMsg] = useState('');`